packages/drivers/routerlicious-driver/src/documentService.ts

```diff
@@ -281,6 +288,14 @@ export class DocumentService
 		// Retry with new token on authorization error; otherwise, allow container layer to handle.
 		try {
 			const connection = await connect();
+			if (
+				(connection as R11sDocumentDeltaConnection).details.supportedFeatures
```

Comment:
This cast is not great. Anything we can do to make it more type-safe?

Reply:
Updated code to also asume details can be undefined by adding a ?